src/wamr/timing.cpp

```diff
@@ -0,0 +1,39 @@
+#include <faabric/util/logging.h>
+#include <wamr/native.h>
+
+#include <stdexcept>
+#include <wasm_export.h>
+
+namespace wasm {
+uint32_t wasi_clock_time_get(wasm_exec_env_t exec_env,
+                             int32_t a,
+                             int64_t b,
+                             int32_t* c)
+{
+    SPDLOG_DEBUG("S - clock_time_get");
+    throw std::runtime_error("clock_time_get not implemented");
+    return 0;
```

Comment:
Same comment re. `return` after `throw` as above, if not necessary there are several places in the PR we can remove.